### ISSUE TITLE
Issue with connection.php.

### ISSUE THREAD

**Issue by VenkSociety** [gauti123456/LoginSystemProject#1]
Warning: mysqli_connect(): (HY000/1045): Access denied for user 'root'@'localhost' (using password: NO) in /home/private/web/puissantroyale.ml/public_html/lib/connection.php on line 2
The connection to the database has not been made

**Comment by JVerhulst** [comment]
did you already figure it out? or do you still need some help?

**Comment by VenkSociety** [comment]
@JVerhulst No, I'm still trying to figure out what to do with the "help" in readme.txt.
